php-practice/section6.php

```diff
@@ -223,7 +270,18 @@ function printErrors(array $errors) {
  *
  * Now make the SAME printErrors function with a foreach loop!
  */
-
-function printErrorsAlternate(array $errors) {
-  // YOUR CODE HERE
+/**I'm going to need your help. I got this to work but I'm not sure why I didn't get the "string conversion error" I normally get.
```

Comment:
I'm not sure now. Before it would tell me that I was trying to convert an array to a string but now it is working fine. Go figure.